cli/azd/internal/appdetect/java.go

```diff
@@ -45,10 +45,13 @@ func (jd *javaDetector) DetectProject(ctx context.Context, path string, entries
 				// we can say that the project is in the root project if the path is under the project
 				if inRoot := strings.HasPrefix(pomFile, rootProject.path); inRoot {
 					currentRoot = &rootProject
+					err := addDockerfileUnderProject(path)
```

Comment:
When multi-module project detected, manually add a Dockerfile in the root dir of sub-module, to skip the pack build (and use normal docker build using Dockerfile).